packages/features/ee/round-robin/roundRobinReassignment.ts

```diff
@@ -0,0 +1,517 @@
+      });
+    }
+
+    await sendRoundRobinCancelledEmails(cancelledRRHostEvt, [
+      {
+        ...previousRRHost,
+        name: previousRRHost.name || "",
+        username: previousRRHost.username || "",
+        timeFormat: getTimeFormatStringFromUserTimeFormat(previousRRHost.timeFormat),
+        language: { translate: previousRRHostT, locale: previousRRHost.locale || "en" },
+      },
+    ]);
+  }
+
+  // Handle changing workflows with organizer
+  if (hasOrganizerChanged) {
+    const workflowReminders = await prisma.workflowReminder.findMany({
+      where: {
+        bookingUid: booking.uid,
+        method: WorkflowMethods.EMAIL,
+        workflowStep: {
+          action: WorkflowActions.EMAIL_HOST,
+          workflow: {
+            trigger: {
+              in: [
+                WorkflowTriggerEvents.BEFORE_EVENT,
+                WorkflowTriggerEvents.NEW_EVENT,
+                WorkflowTriggerEvents.AFTER_EVENT,
+              ],
+            },
+          },
+        },
+      },
+      include: {
+        workflowStep: {
+          include: {
+            workflow: true,
+          },
+        },
+      },
+    });
+
+    for (const workflowReminder of workflowReminders) {
+      const workflowStep = workflowReminder?.workflowStep;
+      const workflow = workflowStep?.workflow;
+
+      if (workflowStep && workflow) {
+        await scheduleEmailReminder({
+          evt: {
+            ...evt,
+            eventType,
+          },
+          action: WorkflowActions.EMAIL_HOST,
+          triggerEvent: workflow.trigger,
+          timeSpan: {
+            time: workflow.time,
+            timeUnit: workflow.timeUnit,
+          },
+          sendTo: reassignedRRHost.email,
+          template: workflowStep.template,
+        });
+      }
+
+      await deleteScheduledEmailReminder(workflowReminder.id, workflowReminder.referenceId);
+    }
+    // Send new event workflows to new organizer
+    const newEventWorkflows = await prisma.workflow.findMany({
+      where: {
+        trigger: WorkflowTriggerEvents.NEW_EVENT,
+        OR: [
+          {
+            activeOn: {
+              some: {
+                eventTypeId: eventTypeId,
+              },
+            },
+          },
+          ...(eventType?.teamId
+            ? [
+                {
+                  activeOnTeams: {
+                    some: {
+                      teamId: eventType.teamId,
+                    },
+                  },
+                },
+              ]
+            : []),
+        ],
+      },
+      include: {
+        steps: {
+          where: {
+            action: WorkflowActions.EMAIL_HOST,
+          },
+          select: {
+            template: true,
+          },
+        },
+      },
+    });
+    for (const workflow of newEventWorkflows) {
+      for (let i = 0; i < workflow.steps.length; i++) {
+        await scheduleEmailReminder({
```

Comment:
Can we use `scheduleWorkflowReminders` for this? `scheduleWorkflowReminders` also takes care of other cases for example to not send workflow notification for not yet confirmed events.